style.css

```diff
@@ -518,3 +538,222 @@ textarea {
   place-items: center;
   margin-top: 20px;
 }
+
+@media screen and (min-width: 768px) {
+  .cont-list {
+    display: flex;
+    gap: 55px;
+    margin-right: 45px;
+    list-style: none;
+    margin-left: 60%;
+  }
+
+  .cont-list li a {
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .nav-logo {
+    margin-left: 10%;
+  }
+
+  /* .introduction p{
+  width: 50%;
+} */
```

Comment:
- Please, let's remove this commented piece of code to keep our codebase clean

Reply:
### Language Image Size

- [ ] Please, on the desktop version, let's kindly improve the size of the list of languages to match the design as our first card is bigger than the remaining two. There are to have the same size. See the image below:

*Figma size*
![Screenshot from 2023-02-23 22-18-44](https://user-images.githubusercontent.com/59397159/221033288-7f7892da-504d-4612-aa4e-7cafb17f9180.png)

*Your size*
![Screenshot from 2023-02-23 22-18-04](https://user-images.githubusercontent.com/59397159/221033296-f762680c-244c-4163-b008-45610a5a092b.png)